### ISSUE TITLE
Chart is not found in cache dir if chart tgz has semver meta

### ISSUE THREAD

**Issue by okgolove** [shini4i/argo-compare#11]
Sonarqube has semver meta info:
https://github.com/SonarSource/helm-chart-sonarqube/releases/tag/sonarqube-4.0.0-sonarqube-dce-3.0.0
```sonarqube-4.0.0+315.tgz```

It seems the app expecting `<chartname>-<version>.tgz` file.

How to reproduce:

1. Create app:
```
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: sonarqube
  namespace: argocd
spec:
  project: default
  source:
    repoURL: https://sonarsource.github.io/helm-chart-sonarqube
    chart: sonarqube
    targetRevision: 3.0.0
    helm:
      values: |
        fullnameOverride: sonarqube-test

  destination:
    name: in-cluster
    namespace: sonarqube
```
2. Apply and push
3. Bump targetRevision to 4.0.0
4. ```argo-compare branch main```

Expected result:
Diff is shown

Actual result:
```
===> Running argo-compare version [v0.0.5]
===> Found the following changed Application files
- sonarqube/sonarqube.yaml
===> Processing changed application: [sonarqube/sonarqube.yaml]
tar: Error opening archive: Failed to open '/Users/user/.cache/argo-compare/https://sonarsource.github.io/helm-chart-sonarqube/sonarqube-4.0.0.tgz'
2023/01/09 14:50:59 exit status 1
```

**Comment by shini4i** [comment]
Thank you for the bug report. I've released a new 0.0.6 version.

Please, let me know if you can reproduce this issue with the new version.

**Comment by okgolove** [comment]
Seems works fine now. Thanks!
